.github/ISSUE_TEMPLATE/intent-to-deprecate--i2d-.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Intent-to-deprecate (I2D)
+about: Proposes deprecating an existing AMP feature.
+title: 'Intent-to-Deprecate: <feature being deprecated>'
+labels: INTENT TO DEPRECATE
+assignees: ''
+
+---
+
+<!--
+Replace/remove all of the text in brackets, including this text.
+
+See https://github.com/ampproject/amphtml/blob/master/spec/amp-versioning-policy.md for details on AMP's deprecation policy, instructions on filling out this I2D template and how to get help if you have questions.
+
+You will file a separate Intent-to-Remove (I2R) after the feature has been deprecated and you are ready to remove the feature.
+-->
+
+## Summary
+<!--
+Provide a brief description of the feature you are planning on deprecating and a concrete plan for deprecating the feature.  Provide an initial plan for removing the deprecated feature; you will file a separate Intent-to-Remove with a detailed removal plan later. 
```

Comment:
…or if the change is ready for immediate removal, rename this to an "Intent to deprecate and remove"

Reply:
SGTM; added.